cmd/capture/main.go

```
@@ -56,53 +30,10 @@ func main() {
-
-	sig := <-quit
-	log.Printf("signal received: %v", sig)
+	srv := server.NewServer(appConfig, nil)
+	log.Println("WARNING: Remember to add http://" + server.GetLocalIP() + ":" + appConfig.Port + "/api/v1/metrics to your Checkmate Infrastructure Dashboard. Without this endpoint, system metrics will not be displayed.")
 
-	ctx, cancel := context.WithTimeout(context.Background(), timeout)
-	defer cancel()
+	srv.Serve()
 
-	return srv.Shutdown(ctx)
+	srv.GracefulShutdown(5 * time.Second)
```

golangci-lint (1.64.8)

Error return value of `srv.GracefulShutdown` is not checked

(errcheck)